src/open_inwoner/templates/pages/category/detail.html

```diff
@@ -17,7 +17,7 @@ <h1 class="h1">
                 {% button icon="edit" text=_("Open in admin") hide_text=True href="admin:pdc_category_change" object_id=object.pk %}
             {% endif %}
         </h1>
-        <p class="p">{{ object.description|linebreaksbr }}</p>
+				{{ object.description|category_ckeditor_content|safe }}
```

Comment:
Please attempt to move this either to the Category model or into the view, ask be in case this doesn't render properly